docs/apache-airflow/authoring-and-scheduling/datasets.rst

```diff
@@ -23,7 +23,7 @@ Data-aware scheduling
 Quickstart
 ----------
 
-In addition to scheduling DAGs based upon time, they can also be scheduled based upon a task updating a dataset.
+In addition to scheduling DAGs based on time, you can also schedule DAGs to run based on when a task updates a dataset.
```

Comment:
Since it's waiting for @sunank200 - comment here to prevent accidental merge :)

Reply:
LGTM